tcmur_cmd_handler.c

```diff
@@ -37,6 +37,10 @@ void tcmur_command_complete(struct tcmu_device *dev, struct tcmulib_cmd *cmd,
 			    int rc)
 {
 	struct tcmur_device *rdev = tcmu_dev_get_private(dev);
+	struct tcmur_cmd_state *rcmd_state = tcmu_cmd_get_private(cmd);
+
+	if (rcmd_state)
+		free(rcmd_state);
```

Comment:
Here is one problem.
The cmd is allocated by using malloc(), so the cmd->hm_private maybe any possible random value. And if the command is any of the sync type which falls in handle_sync_cmd() will it crash here ?

Reply:
Yeah, i think you're right. Could we just change malloc into calloc for tcmulib_cmd allocation to fix this?

Reply:
Yeah, that's okay for me.